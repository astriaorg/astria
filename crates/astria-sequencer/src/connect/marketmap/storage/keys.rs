pub(in crate::connect::marketmap) const MARKET_MAP: &str = "connect/market_map/market_map";
pub(in crate::connect::marketmap) const PARAMS: &str = "connect/market_map/params";
pub(in crate::connect::marketmap) const LAST_UPDATED: &str = "connect/market_map/last_updated";

#[cfg(test)]
mod tests {
    use super::*;

    const COMPONENT_PREFIX: &str = "connect/market_map/";

    #[test]
    fn keys_should_not_change() {
        insta::assert_snapshot!("market_map_key", MARKET_MAP);
        insta::assert_snapshot!("params_key", PARAMS);
        insta::assert_snapshot!("last_updated_key", LAST_UPDATED);
    }

    #[test]
    fn keys_should_have_component_prefix() {
        assert!(MARKET_MAP.starts_with(COMPONENT_PREFIX));
        assert!(PARAMS.starts_with(COMPONENT_PREFIX));
        assert!(LAST_UPDATED.starts_with(COMPONENT_PREFIX));
    }
}
