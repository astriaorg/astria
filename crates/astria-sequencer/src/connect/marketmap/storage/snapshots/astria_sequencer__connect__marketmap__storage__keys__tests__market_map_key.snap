---
source: crates/astria-sequencer/src/connect/marketmap/storage/keys.rs
expression: MARKET_MAP
---
connect/market_map/market_map
